WhisperAPI/WhisperAPI/Controllers/SuggestionsController.cs

```diff
@@ -27,6 +32,19 @@ public SuggestionsController(ISuggestionsService suggestionsService, IQuestionsS
         [HttpPost]
         public IActionResult GetSuggestions([FromBody] SearchQuery searchQuery)
         {
+            var currentDetectedModulesAndMatchScore = new ModuleDetector().DetectModuleList(searchQuery.Query);
```

Comment:
ModuleDetector doesn't need to be instanciated every time since it is stateless, could be part of the current class